### PR TITLE
ci: Fix flaky Happo tests

### PR DESCRIPTION
**Pull request by Igarfinkle**
# Summary

Attempts to fix remaining flaky happo tests, especially in Tooltip.

### CODE REVIEW

File: src/components/forms/DatePicker/DatePicker.stories.tsx [hunk 1]
```diff
@@ -92,7 +92,7 @@ export const withDefaultInvalidValue = (): React.ReactElement => (
 )
 withDefaultValue.parameters = {
   happo: {
-    waitForCOntent: '1988-05-16',
+    waitForContent: '1988-05-16',
```
Comment (brandonlenz) [inline src/components/forms/DatePicker/DatePicker.stories.tsx:95]: Oh dang. Nice catch!

File: src/components/Tooltip/Tooltip.stories.tsx [hunk 2]
```diff
@@ -126,6 +133,14 @@ export const tooltipBottomLeftWrap = (): React.ReactElement => (
     </Tooltip>
   </div>
 )
+tooltipBottomLeftWrap.parameters = {
+  happo: {
+    waitFor: () =>
+      document.querySelector(
+        '.usa-tooltip__body.usa-tooltip__body--right.usa-tooltip__body--wrap.is-visible.is-set'
```
Comment (brandonlenz) [inline src/components/Tooltip/Tooltip.stories.tsx:140]: Depending on the screen size that happo is taking a screenshot for, the tooltip position may use a different default position, which would mean different different classes than what you expect to find. That probably limits us to doing something like:

```suggestion
        '.usa-tooltip__body.usa-tooltip__body--wrap.is-visible.is-set'
```

Or more generically using waitFor at the top level with `'.usa-tooltip__body.is-visible.is-set'`.

A lot of the flaky diffs _are_ on the ios-safari screenshots, which usually forces the alternate position of the tooltip.

**Review by brandonlenz — CHANGES_REQUESTED** [review]

File: src/components/Tooltip/Tooltip.stories.tsx (hunk 2 above)
Comment (Igarfinkle) [reply src/components/Tooltip/Tooltip.stories.tsx]: Fixed!

**Review by brandonlenz — APPROVED** [review]

**Comment by brandonlenz** [comment]
I've permanently ignored the ios-safari diffs for tooltip. Happo seems to crop the screenshots differently every time, despite now waiting for the tooltip to be visible before taking them. We should still merge this as it *should* in theory make these diffs more stable

**Pull request merged**